routes/api-routes.js

```diff
@@ -36,6 +36,10 @@ module.exports = (app) => {
         res.json(req.user);
     });
 
+    app.post("/api/logi", passport.authenticate("local"), function(req, res) {
```

Comment:
This was just for testing purposes (api/logi misspelled login) and should be removed